### ISSUE TITLE
Ensure that styles.css only includes imports

### ISSUE THREAD

**Issue by sleicester** [the-open-university/ouice#25]
_From @sleicester on May 20, 2016 17:21_

_Copied from original issue: the-open-university/student-theme-framework/pull/25_

**Comment by stuartcrouch** [comment]
Devil's advocate here. Shouldn't we be using gulp to merge and minify the CSS  and JS so that poor mobile users don't have to download all those extra bytes?

**Comment by bgurney** [comment]
I think this will be up to the implementation to do this? I believe that OU ICE is just CSS (Less, Sass) and HTML.

**Comment by sleicester** [comment]
Yep, completely agree and this is the end goal. Making available a concatenated and minified version as well as the components that you could pull together yourself.

It's probably worth including a Gulpfile as part of the repository to make it easy for anyone cloning to do this as well.
